Source/WTF/wtf/WeakHashSet.h

```diff
@@ -188,6 +188,8 @@ class WeakHashSet final {
         return result;
     }
 
+    bool computeIsEmpty() const { return !computeSize(); }
```

Comment:
This seems redundant with `isEmptyIgnoringNullReferences()` no?

Reply:
IgnoringNullReferences will count non-zero if an object has been freed but its weakptr hasn’t been explicitly removed from the table yet. This compute alternative rules out stale null entries.

Reply:
> IgnoringNullReferences will count non-zero if an object has been freed but its weakptr hasn’t been explicitly removed from the table yet.

Either I'm missing something or this statement is not accurate. As far as I can tell, they both do the same thing.

Reply:
Oops! Yeah, I got confused by the iterator syntax, and I couldn’t tell from the name whether ‘ignore’ would mean zero or non-zero.